jobs/calculate_users_response.js

```diff
@@ -0,0 +1,73 @@
+const { Order } = require('../models');
```

Comment:
All files in jobs have functions that are executed as cronjobs, so this shouldn't be here